colaraz/cms/templates/widgets/header.html

```diff
@@ -134,21 +134,24 @@ <h3 class="title"><span class="label">${_("Tools")}</span> <span class="icon fa
         user_urls = user.colaraz_profile.role_based_urls
         colaraz_notifications_url = user_urls.get('viewNotifications') if user_urls.has_key('viewNotifications') else '#'
 
-        invite_friend_url = '{}://{}.{}/Home/InviteFriends.aspx'.format( protocol, identifier, domain)
+        invite_friend_url = '{}://{}.{}/{}'.format(protocol, identifier, domain, user_urls.get('inviteFriend', '#'))
         see_all_notifications_url = '{}://{}.{}/{}'.format( protocol, identifier, domain, colaraz_notifications_url)
+        colaraz_ecosystem_url = '{}://{}.{}/cvsocial/activity?{}'.format(protocol, identifier, domain, user_urls.get('ecosystem', '#'))
+        
       %>
       <nav class="nav-account nav-is-signedin nav-dd ui-right" aria-label="${_('Account')}">
         <h2 class="sr-only">${_("Account Navigation")}</h2>
         <ol>
           <li class="dropdown">
-            <a class="nav-link" href="#" onclick="removeNotificationsCount();" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            <a class="nav-link" href="#" onclick="removeNotificationsCount();loginToEcosystem('${colaraz_ecosystem_url}');" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
instead of adding two function calls here, let's add a single callback here and call these two functions from there.

Reply:
Both functions have different tasks to get done, and they are not interlinked. So, I don't think we should go the callback way.

Reply:
Updated.